src/tools/slimetora.md

```diff
@@ -51,24 +63,38 @@ SlimeTora is a program that connects the HaritoraX trackers to the SlimeVR serve
+The most likely cause for this is an incorrect calibration - try to redo the calibration steps in SlimeVR to improve the results. The following video by ZRock35 is a great guide on how to get the best results for SlimeVR calibration: [https://www.youtube.com/watch?v=SYqfQdVseF4](https://www.youtube.com/watch?v=SYqfQdVseF4)
+
+By default, SlimeVR uses the `prediction` filtering setting to estimate where your trackers will be, so switching from it to `smoothing` or `no filtering` may also help with the trackers being out-of-place (unconfirmed).
+
+If you are experiencing issues specifically with lower-body, it is still usually a calibration issue so as mentioned above, try to redo the calibration. You may also try disabling the knees in SlimeVR's settings under `SteamVR Trackers` to see if it helps (disable `Automatic tracker assignment` then disabling both knees). The knee trackers will still be used for the skeletal tracking in SlimeVR but not appear as extra trackers in SteamVR, which _may_ help with some tracking issues in your lower-body.
 
 ### My trackers are not aligned correctly to my body!
 
-Make sure you've done the `Body proportions calibration` correctly, either manually or automatically, and also make sure you have done `Automatic mounting calibration`, **NOT** manual. You can also change settings in the game you are running (e.g. VRChat's IK settings like `Legacy calibration`) or change the positions of the trackers physically.
+Make sure you've done the `Body proportions calibration` correctly, either manually or automatically, and also make sure you have done `Automatic mounting calibration` (as manual mounting is quite finicky with these trackers). You can also change settings in the game you are running (e.g. VRChat's IK settings like `Legacy calibration`) or change the positions of the trackers physically.
 
 If running VRChat, there are some recommended settings for it [here](./vrchat-config.md).
 
-*Written by JovannMC, software developed by BracketProto & JovannMC*
+### "Automatic proportions" is disabled/trackers won't show up in SteamVR!
+
+Assuming of course you have your VR headset on at this stage, make sure the SlimeVR add-on in SteamVR has been installed/enabled correctly. You can check this by looking for `slimevr` in SteamVR settings > Manage Add-ons and enable if it disabled. If it is not installed correctly and isn't showing up, try reinstalling the SlimeVR server.
+
+You may also install the SlimeVR add-on manually by downloading the driver from the [GitHub repo](https://github.com/SlimeVR/SlimeVR-OpenVR-Driver/tree/main) (Code > Download ZIP), and extracting the contents of the `/driver/slimevr` folder into `C:\Program Files (x86)\Steam\steamapps\common\SteamVR\drivers\slimevr` for Windows or `~/.steam/root/steamapps/common/SteamVR/drivers/` for Linux (create the `slimevr` directory if it doesn't exist).
```

Comment:
This would be nice to have as a general common issue, though the instructions for the driver are wrong, you need to download the release, not the code. The code won't have the compiled binary lol. I do have some instructions at https://bscotch.ca/vr/slimevr-manual-driver-check.html for reference.